### DISCUSSION TITLE
Welcome to StockOfSentence Discussions!

### DISCUSSION THREAD

**Discussion by Subham-Maity** [Subham-Maity/StockOfSentence#48] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pragyaasapkota** [comment]
It might be better if the added quotes are separated in the categories of some kinds like motivation - general - and such.

**Reply by Subham-Maity** [reply]
> It might be better if the added quotes are separated in the categories of some kinds like motivation - general - and such.

@Pragya2056 Could you please create an issue with some details so that I can attach labels to it

**Reply by pragyaasapkota** [reply]
sure I will create an issue with some details.

**Comment by pragyaasapkota** [comment]
@Subham-Maity 
I have created issue #126 in response to my comment.
